src/tools/sssctl/sssctl_config.c

```diff
@@ -93,7 +98,10 @@ errno_t sssctl_config_check(struct sss_cmdline *cmdline,
     }
 
     if (config_path != NULL) {
```

Comment:
One question - right now config_snippet_path can't exist without config_path set. If this is by design? If the config_snippet_path should be always subdirectory of config_path and that is the reason why it can't exist without it?

Reply:
good catch, I did not think about such use-case but it is valid one. fixed.